webapp/javascript/components/TimelineChart/TimelineChartSelection.ts

```diff
@@ -116,8 +117,14 @@ const handleHeight = 22;
       return isLeftSelecting ? 'left' : isRightSelecting ? 'right' : null;
     }
 
-    function setCursor(type: string) {
-      $('canvas.flot-overlay').css('cursor', type);
+    function cursor() {
+      let current = '';
+      return function (type: string) {
+        if (type && current !== type) {
+          current = type;
+          $('canvas.flot-overlay').css('cursor', type);
```

Comment:
What if there are 2 canvas in the same page?

Reply:
I wonder if there isn't a way to get the actual element we care about cc @pavelpashkovsky